sidecar/src/figwheel_sidecar/system.clj

```diff
@@ -130,7 +131,13 @@
                    channel-id msg-data callback))
   (-connection-data [this]
     (server/-connection-data (:figwheel-server @system)))
-  (-actual [this] (:figwheel-server @system)))
+  (-actual [this] (:figwheel-server @system))
+  Suspendable
+  (suspend [c] c)
+  (resume [c old-c]
```

Comment:
I think you want to check if it was suspended, and then assoc the values on, and otherwise start the component. See https://github.com/weavejester/ring-jetty-component/blob/87ac67ebc1d9cbc8fd340c880e5bc9a2fd8ac392/src/ring/component/jetty.clj#L25-L40 for a more complete suspendable implementation.

I don't think this will resume properly in all cases.